app/scripts/modules/core/src/pipeline/config/pipelineConfigProvider.ts

```diff
@@ -107,7 +109,12 @@ export class PipelineConfigProvider implements IServiceProvider {
         }
       });
     } else {
-      cloudProviders = providers;
+      cloudProviders = Array.from(new Set(accounts.reduce((memo, acc) => {
```

Comment:
what's the benefit of converting an array to a set and back to an array again?

Reply:
The intention is to ensure each element in the array is unique.

Reply:
Ah, got it.

Reply:
updated to use the more explicit `uniq` lodash helper.